Sources/Components/TextField/TextField.swift

```diff
@@ -162,15 +167,16 @@ public class TextField: UIView {
             textField.topAnchor.constraint(equalTo: typeLabel.bottomAnchor, constant: .mediumSpacing),
             textField.leadingAnchor.constraint(equalTo: leadingAnchor),
             textField.trailingAnchor.constraint(equalTo: trailingAnchor),
-            textField.trailingAnchor.constraint(equalTo: trailingAnchor),
```

Comment:
Uups.. :)

Reply:
Hehe, yeah noticed that it was set twice. Changed the last one to be bottom anchor. As the underline height now changes I didn't want that one to be the one attached to the bottom. That would cause the whole view to resize when activating the textfield.

Reply:
Yeah, that was probably a brain fart from my part. Hence, the "Uups.." ;)

Reply:
![giphy 12](https://user-images.githubusercontent.com/1088217/37346731-6aadea08-26d0-11e8-8486-1e04eab1663c.gif)